.github/workflows/BaselineProfile.yaml

```
@@ -0,0 +1,54 @@
+name: Create new baseline profile
+
+on:
+  # every day at 00:23
+  schedule:
+    - cron: '23 0 * * *'
+  workflow_dispatch:
+
+jobs:
+  baseline-profile:
+    runs-on: macos-latest
+    timeout-minutes: 45
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3   
+
+      - name: Validate Gradle Wrapper
+        uses: gradle/wrapper-validation-action@v1
+
+      - name: Copy CI gradle.properties
+        run: mkdir -p ~/.gradle ; cp .github/ci-gradle.properties ~/.gradle/gradle.properties
+
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'zulu'
+          java-version: 11
+
+      - name: Build app
+        run: ./gradlew assembleBenchmark
```

this will build both demo and prod .. I think we can run just `assembleDemoBenchmark` for benchmarks
```suggestion
        run: ./gradlew assembleDemoBenchmark
```